test/unit/memcached_test.rb

```diff
@@ -1,524 +0,0 @@
-require File.expand_path("#{File.dirname(__FILE__)}/../test_helper")
```

Comment:
Did you mean to delete this entire test file?

Reply:
Yes. It's subtle but it is actually dead code.

That was part of my confusion.

Reply:
Oh, because the base class was removed from the test class